src/DependencyInjection/SetonoSyliusGiftCardExtension.php

```diff
@@ -15,13 +15,39 @@ final class SetonoSyliusGiftCardExtension extends AbstractResourceExtension
     public function load(array $configs, ContainerBuilder $container): void
     {
         /**
-         * @var array{code_length: int, driver: string, resources: array<string, mixed>} $config
+         * @var array{
+         *     pdf_rendering: array{
+         *         default_orientation: string,
+         *         available_orientations: array<array-key, string>,
```

Comment:
```suggestion
         *         available_orientations: list<string>,
```
I have started to use `list` where applicable just to get used to it. It adds another layer of static analysis and hence safety :)